server/server.py

```diff
@@ -85,16 +85,30 @@ def parseArgs():
 	parser = argparse.ArgumentParser(description='Location logging server')
 
 	parser.add_argument('-a', '--address', metavar='ADDRESS', type=ip_address)
+	parser.add_argument('-c', '--configure', action='store_true')
 	parser.add_argument('-p', '--port', metavar='PORT', type=int)
 
 	return parser.parse_args()
 
 def main():
-	HOST, PORT = "0.0.0.0", 3145
-
-	args = parseArgs()
+	mysettings = Settings()
+	#Load settings from file
+	if !(mysettings.loadSettings()):
+		print("[-] Can't open configuration settings, Exiting")
+		exit(2)
+	HOST = mysettings.getSetting('Address')
+	PORT = mysettings.getSetting('Port')
+	logger = configDebugLog(mysettings.getSetting('LogFile'))
```

Comment:
After looking at your fix, it got me thinking. Have you tested to see if the logger is working in functions defined in stcommon? Specifically the functions that are called by client or server? Technically logger should be in a different namespace than the one defined in client or server. Python is different from C in that includes don't just dump the obj's together and let the linker figure out the references.